Collectors/wlcg_converter.py

```diff
@@ -199,6 +199,20 @@ def ConvertGstream(source_record):
 
     return source_record
     
+def ConvertGstreamTPC(source_record):
```

Comment:
This function looks almost the exact same as the function above.  Would it be easier to have a single function, but pass in the "producer" as an argument to the function?

Reply:
sure